helpers/postgresql.py

```diff
@@ -39,6 +35,7 @@ def parseurl(url):
 
 
 class Postgresql:
+    _SERVER_VERSION = 90400
```

Comment:
In issue #18 it is already mentioned that we might be able to support 9.3, can we make this a non-magic variable?